Migrate CI to dda (#33587)
DataDog/datadog-agent: pkg/proto/datadog/api/README.md

## gRPC: Protobuf and Gateway code generation

To generate the code for the API you have defined in your `.proto`

> files we will need three different grpc-related packages:

Reviewer: ```suggestion
files requires three different grpc-related packages:
```

From the repository root run the following:

```
deva inv protobuf.generate
```

> Those two will generate the protobuf golang definitions _and_ the

Reviewer: ```suggestion
This command generates the protobuf golang definitions _and_ the
```

> gRPC gateway code that will allow us to serve the API also as a

Reviewer: ```suggestion
gRPC gateway code that allows Datadog to serve the API also as a
```
